calculatedmetrics.md

```diff
@@ -1,10 +1,287 @@
 
+To obtain authorization and authentication, see the [Getting Started Guide for 2.0 APIs](https://www.adobe.io/apis/experiencecloud/analytics/docs.html#!AdobeDocs/analytics-2.0-apis/master/readme.md).
+
+## /calculatedmetrics endpoint description
+
+The `/calculatedmetrics` endpoint description is shown in our [Swagger UI](https://adobedocs.github.io/analytics-2.0-apis/). Use the Swagger UI to see endpoint summaries, available methods, parameters, example values, models, and status codes, and to try out the API.
+
+## Best practices
+
+Please follow these guidelines when using the Calculated Metrics APIs:
+
+*  Make multiple requests, each one limited to 1000 calculated metrics or fewer when retrieving them. Avoid making a large, single request for all possible calculated metrics.
+*  Request data once and cache it.
+*  Try to avoid creating duplicate calculated metrics with the same definition. Creating many calculated metrics will affect performance for your company in some situations.
+*  Creating a calculated metric from scratch can be difficult without familiarity of the definition terms. We recommend using the Analysis Workspace UI for creating a calculated metric. Then use this API to retrieve the calculated metric and modify the definition to match what you want to do programmatically. Then we recommend that you `POST` that calculated metric with the modified definition back to this API.
+
+## Predefined calculated metrics
+
+Adobe provides predefined calculated metrics templates for clients. Although original templates cannot be modified or deleted, copies of them can be made and then the copies can be modified. You can identify templates by the additional attribute `template: true`. Also, all template ids start with `cm_`. These calculated metrics templates can be used in Workspace projects like any other type of calculated metric but will not work in any other contexts.
+
+## Expansions
+
+The primary calculated metrics endpoints support the URL query parameter `expansion`. This parameter allows requests for some response fields to be populated that would otherwise not be included. The available options for expansion can be viewed in our [Swagger UI](https://adobedocs.github.io/analytics-2.0-apis/). They include the following:
+
+* `reportSuiteName`
+* `siteTitle`
+* `ownerFullName`
+* `modified`
+* `isDeleted`
+* `approved`
+* `favorite`
+* `shares`
+* `tags`
+* `sharesFullName`
+* `usageSummary`
+* `usageSummaryWithRelevancyScore`
+* `definition`
+* `authorization`
+* `compatibility`
+* `categories`
+* `reportTimeAttribution`
+* `warning`
+
+## IncludeTypes
+
+The GET multiple `/calculatedmetrics` endpoint supports the query parameter `includeType`. This parameter alters the set of calculated metrics that are included in API responses. By default, the response with this parameter includes only the calculated metrics owned by the user making the request. By using this parameter, the following values are also possible:
```

Comment:
By default, the response (i.e. _without_ this parameter) includes only the calc metric owned by the user.

When using this parameter, the following _options_ are possible: